File: application/localvision/samples/general/example1.f
     90 evaluate:              #--  run the program asof 1990
        [
        !top3 <- Company masterList           #-- start with all companies
            rankDown: [ :sales pctChangeLag: 1 yearEnds ] .  #-- 1990 to 1989
            select: [ rank <= 3 ] ;           #-- select rank = 1-3
     
        "***  Financial Summary Report For Top 3 Companies  ***" printNL ;
        "       (Using 1990-1989 Percent Change in Sales)" printNL ;
     
        top3                                  #-- for each top3 company
        do: [ newLine print ; 
              newLine print ; 
              financialAnalysis ;             #-- run the method
            ] ;
        ] ;

File: application/localvision/samples/general/example1.c
     Named Company GM
     do: [ 
         !dr <- 90 to: 88 by: 1 yearEnds ;
         name center: 45 . printNL ;
         "Financial Summary" center: 45 . printNL ;
         "Prepared: " print ;
         ^today formatUsingShortName print ;      #-- format today's date
         "     Latest Price: " print ;            #-- display string
         price printNL ;                          #-- display price
         newLine print ; 
     
         " " print: 15 ; 
         dr evaluate: [ ^date year print: 10 ; ] ;
         newLine print ; 
         " " print: 15 ;
         dr evaluate: [ " " print ; "-" fill: 9 . print ] ;
         newLine print ; 
     
         "Sales" print: 15 ;
         dr evaluate: [ sales print: 10 ] ;
         newLine print ;
     
         "EPS" print: 15 ;
         dr evaluate: [ earningsPerShare print: 10 ] ;
         newLine print ;
     
         "PE" print: 15 ;            #-- PE is a calculation
         dr evaluate: [ price / earningsPerShare print: 10 ] ;
         newLine print ;
         ] ;
        


